Screens/AlbumDetailScreen.swift

```diff
@@ -86,29 +86,33 @@ struct AlbumDetailScreen: View {
                         .foregroundColor(.accentColor)
                 )
 
-            Button {
-                // Album shuffle play action
+            AsyncButton {
+                let songs = library.getSongs(for: album)
+                try await player.play(songs: songs.shuffled())
```

Comment:
This could be further improved by moving into play button (`shuffle: bool`), but then the button also needs to accept custom label first (only accepts text). But this is more like a note as modifying play button would unnecessarily increase scope.

So just to keep something in mind - https://github.com/lyarenei/Medusic/issues/86#issue-2259874412